apluslms_roman/builder.py

```diff
@@ -4,10 +4,13 @@
 from apluslms_yamlidator.utils.decorator import cached_property
 from apluslms_yamlidator.utils.collections import OrderedDict
 
+from apluslms_roman.utils.path_mapping import get_host_path
```

Comment:
You are importing with this absolute path and with a relative path below from the same module `path_mapping`.